openreview/api/client.py

```diff
@@ -2885,6 +2894,12 @@ def to_json(self):
         if self.id:
             body['id'] = self.id
 
+        if self.invitations:
```

Comment:
is there a reason to add "invitations" here? I don't think the API allows posting an invitation with the property invitations.

Reply:
However, when you get an invitation it had the `invitations` field. So I think this field is necessary in the Invitation class

Reply:
It can be in the class but I ignore them in then to_json function that is where we convert the object to a JSON to send to the API.